src/index.js

```diff
@@ -43,8 +43,12 @@ const prettierOptions = [
 function getOptions(array) {
   const options = {};
   array.forEach(([flag]) => {
-    const key = camelCase(flag);
-    options[key] = program[key];
+    const key = camelCase(flag.replace(' <int>', ''));
+    let value = program[key];
+    if(typeof value === 'string') {
+      value = parseInt(value)
+    }
```

Comment:
This should be.
```javascript
    if (/<int>/.test(flag) && typeof value === 'string') {
      value = parseInt(value, 10);
    }
```
 Otherwise, i think it will try to parse `--parser <flow|babylon>` as an int.